Release 1.0.0 (#51)
plotly/dash-player: CHANGELOG.md

# Change Log for Dash Player

All notable changes to `dash-player` will be documented in this file.
This project adheres to [Semantic Versioning](https://semver.org/).

---

## [1.0.0] - 2022-10-20

> This release is `v1.0.0` as it that `dash-player` is now production-ready.

Reviewer: ```suggestion
This release is `v1.0.0` to indicate that `dash-player` is now considered production-ready.
```
